assemblyline/common/identify.py

```diff
@@ -43,39 +43,39 @@
     'code/csharp': [
         re.compile(rb'(^|\n)[ \t]*namespace[ \t]+[\w.]+'),
         re.compile(rb'(^|\n)[ \t]*using[ \t]+[\w.]+;'),
-        re.compile(rb'(^|\n)[ \t]*internal class '),
+        re.compile(rb'(^|\n)[ \t]*internal[ \t]+class[ \t]+'),
     ],
     'code/php': [
         re.compile(rb'(^|\n)<\?php'),
         re.compile(rb'namespace[ \t]+[\w.]+'),
-        re.compile(rb'function[ \t]*\w+[ \t]*\(\$[^)]+\)[ \t]*{'),
+        re.compile(rb'function[ \t]+\w+[ \t]*\([ \t]*\$[^)]+\)[ \t]*{'),
```

Comment:
\n could be added to the whitespace wildcard before the { to allow for alternate brace styles